susemanager/susemanager.changes

```diff
@@ -1,3 +1,4 @@
+- add support for in-place migration
```

Comment:
Since in-place migration is not supported for SUSE Manager, I think we should mention this as Uyuni-Only (and of course we can revisit this later).

Reply:
In the longer run we also want this for SUSE Manager. But I'm fine restricting it to Uyuni for the time being.